.github/renovate.json

```diff
@@ -18,18 +18,26 @@
       "matchUpdateTypes": ["minor", "patch"]
     },
     {
-      "groupName": "web npm deps non-major",
       "matchManagers": ["npm"],
-      "matchUpdateTypes": ["minor", "patch"],
       "matchFileNames": ["web/package.json"],
       "addLabels": ["ui"]
     },
+    {
+      "matchManagers": ["npm"],
+      "matchFileNames": ["docs/**/package.json"],
+      "addLabels": ["documentation"]
+    },
+    {
+      "groupName": "web npm deps non-major",
```

Comment:
This is the same rule as for docs non-major now. Use just one role with multiple matchFilenames?

Reply:
Sure but why would you keep a different group name? These group names are nor used elsewere so no grouping happens at all and we can just use a more generic group name as description.

Reply:
Im aware of it it just thought it doesnt matter if the npm non-major pr is for web or docs. If you want to keep it fine for me as well.